src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -73,6 +74,334 @@ def function(self, optimization_problem, ensemble_member):
         return optimization_problem.state(self.state)
 
 
+class EndScenarioSizingDiscountedHIGS(
```

Comment:
I would prefer to keep the same structure as we have for other problem definitions.
A main class which inherits all the Mixins and the problem and then a separate class to change the solversettings to HIGHS, which inherits from the main class

Reply:
done